src/foam/u2/ActionView.js

```diff
@@ -92,9 +118,16 @@ foam.CLASS({
       if ( this.icon ) {
         // this.nodeName = 'a';
         this.start('img').attr('src', this.icon).end();
-      }
-      if ( this.showLabel )
+      } else if (this.iconFontName) {
+        this.nodeName = 'i';
+        this.cssClass(this.action.name);
+        this.cssClass(this.iconFontClass); // required by font package
+        this.style({'font-family': this.iconFontFamily});
+        this.add(this.iconFontName);
+        this.setAttribute('title', this.action.toolTip); // hover text
```

Comment:
Is the tooltip dependent on the iconFontName being set? Can't it work for the other cases also?